docs/components/sensor/ds18b20.md

```diff
@@ -60,3 +60,13 @@ The following attributes are available for `ds18b20` sensors:
 ## Test the sensor
 
 {{< readfile "/static/include/components/test-control/sensor-control.md" >}}
+
+## Next steps
+
+The [sensor API](/appendix/apis/components/sensor/) or check out one of these guides:
```

Comment:
```suggestion
Check out the [sensor API](/appendix/apis/components/sensor/) or check out one of these guides:
```
something like this might make sentence less awk

Reply:
if you apply do same to below!